akka-docs/src/main/paradox/persistence.md

```diff
@@ -30,12 +30,12 @@ to see what this looks like in practice.
 
 ## Introduction
 
-Akka persistence enables stateful actors to persist their internal state so that it can be recovered when an actor
-is started, restarted after a JVM crash or by a supervisor, or migrated in a cluster. The key concept behind Akka
-persistence is that only changes to an actor's internal state are persisted but never its current state directly
-(except for optional snapshots). These changes are only ever appended to storage, nothing is ever mutated, which
-allows for very high transaction rates and efficient replication. Stateful actors are recovered by replaying stored
-changes to these actors from which they can rebuild internal state. This can be either the full history of changes
+Akka persistence enables stateful actors to persist their state so that it can be recovered when an actor
+is either restarted after a JVM crash or by a supervisor, or migrated within a cluster. The key concept behind Akka
+persistence is that only the _messages_ received by the actor are persisted, not the actual state of the actor
```

Comment:
It should not be the messages it receives (that'd be command sourcing) it is events representing state changes.

Reply:
Oh right. lol